desktop/src/@batch-flask/ui/tabs/tabs.scss

```diff
@@ -0,0 +1,9 @@
+@import "app/styles/variables";
+
+mat-tab-group {
+    &:not(.form-tabs) .mat-tab-label.cdk-keyboard-focused {
+        border-width: 1px;
+        border-style: solid;
+        border-color: $button-basic-text-color !important;
```

Comment:
Would there be another way to achieve this border-color without adding `!important`? I believe it's better to avoid using this rule if possible since it could potentially override other styles.